hrldas/HRLDAS_forcing/run/examples/ERA5/namelist.input.ERA5

```diff
@@ -0,0 +1,82 @@
+&files
```

Comment:
@CharlesZheZhang Looks like the FORCING_TYPE entry is missing in this namelist?

Reply:
I will add this line in the namelist.